draft-ietf-quic-reliable-stream-reset.md

```diff
@@ -171,6 +171,28 @@ bit for the same stream, the initiator MUST NOT change the Application Error
 Code or the Final Size. If the receiver detects a change in those fields, it
 MUST close the connection with a connection error of type STREAM_STATE_ERROR.
 
+# Implementation Guidance
+
+Semantically, the CLOSE_STREAM frame is more akin to the FIN bit than to the
+RESET_STREAM frame.
```

Comment:
Maybe say it can be thought of as a combination of the two?

Reply:
Considering that this is the intro leading to the advice that CLOSE_STREAM should be implemented reusing code to handle FIN, I think we want to say that the frame has more commonality with FIN than RESET_STREAM.

I think the problem might be due to the use of the term"semantically," I've changed that to "In terms of transport machinery" in commit de5fe8b.

PTAL.